scripts/release/update-comment.js

```diff
@@ -1,27 +1,35 @@
-#!/usr/bin/env node
```

Comment:
`version.sh` executes this file without specifying `node`, so this and the executable bit are necessary unless we update that script. `npm version` is currently failing because of that.